### PR TITLE
DOC: ecosystem.md: add pygwalker, add seaborn code example

### PR DESCRIPTION
**Pull request by westurner**
- [ ] closes #xxxx (Replace xxxx with the GitHub issue number)
- [ ] [Tests added and passed](https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#writing-tests) if fixing a bug or adding a new feature
- [ ] All [code checks passed](https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#pre-commit).
- [ ] Added [type annotations](https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#type-hints) to new arguments/methods/functions.
- [ ] Added an entry in the latest `doc/source/whatsnew/vX.X.X.rst` file if fixing a bug or adding a new feature.

### CODE REVIEW

**Comment by westurner** [comment]
- https://github.com/Kanaries/pygwalker

File: web/pandas/community/ecosystem.md [hunk 1]
```diff
@@ -82,6 +82,20 @@ pd.set_option("plotting.backend", "pandas_bokeh")
 It is very similar to the matplotlib plotting backend, but provides
 interactive web-based charts and maps.
 
+### [pygwalker](https://github.com/Kanaries/pygwalker)
+
+PyGWalker is an interactive data visualization and 
+exploratory data analysis tool built upon Graphic Walker
```
Comment (mroeschke) [inline web/pandas/community/ecosystem.md:88]:
```suggestion
PyGWalker is an interactive data visualization and
exploratory data analysis tool built upon Graphic Walker
```

**Review by mroeschke — APPROVED** [review]

**Comment by mroeschke** [comment]
Thanks @westurner

**Comment by westurner** [comment]
Thanks!

**Pull request merged**